src/atomate2/vasp/jobs/core.py

```diff
@@ -456,6 +455,45 @@ class DielectricMaker(BaseVaspMaker):
     )
 
 
+@dataclass
+class PolarizationMaker(BaseVaspMaker):
+    """
+    Maker to create polarization calculation VASP jobs.
+
+    .. Note::
+        If starting from a previous calculation, magnetism will be disabled if all
+        MAGMOMs are less than 0.02.
+
+    Parameters
+    ----------
+    name : str
+        The job name.
+    input_set_generator : .StaticSetGenerator
+        A generator used to make the input set.
+    write_input_set_kwargs : dict
+        Keyword arguments that will get passed to :obj:`.write_vasp_input_set`.
+    copy_vasp_kwargs : dict
+        Keyword arguments that will get passed to :obj:`.copy_vasp_outputs`.
+    run_vasp_kwargs : dict
+        Keyword arguments that will get passed to :obj:`.run_vasp`.
+    task_document_kwargs : dict
+        Keyword arguments that will get passed to :obj:`.TaskDocument.from_directory`.
```

Comment:
The task document was refactored to emmet so you can update this.
```suggestion
        Keyword arguments that will get passed to :obj:`.TaskDoc.from_directory`.
```

Reply:
done